src/makeCalculator.js

```diff
@@ -4,7 +4,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const operate = (callback, num) => {
+    result = callback(result, num);
+
+    return calculator;
+  };
+
+  const add = (prevResult, num) => {
+    return prevResult + num;
+  };
+
+  const subtract = (prevResult, num) => {
+    return prevResult - num;
+  };
+
+  const multiply = (prevResult, num) => {
+    return prevResult * num;
+  };
+
+  const divide = (prevResult, num) => {
```

Comment:
You need to add a check for 0, because you cannot divide by 0 according to the rules of mathematics